docs/data/material/components/snackbars/snackbars.md

```diff
@@ -9,109 +9,88 @@ waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
 
 # Snackbar
 
-<p class="description">Snackbars provide brief notifications. The component is also known as a toast.</p>
-
-Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn't interrupt the user experience, and they don't require user input to disappear.
-
-Snackbars contain a single line of text directly related to the operation performed.
-They may contain a text action, but no icons. You can use them to display notifications.
+<p class="description">Also known as a toast, snackbars are used for brief notifications of a process that has been performed or will be performed.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-**Frequency**: Only one snackbar may be displayed at a time.
+## Introduction
 
-## Simple snackbars
+```jsx
+import Snackbar from '@mui/material/Snackbar';
+```
 
-A basic snackbar that aims to reproduce Google Keep's snackbar behavior.
+The Snackbar provides users updates on an app's processes, temporarily appearing on the screen.
```

Comment:
I think "floats above the UI" and "non-critical updates" are important details here for differentiating a Snackbar from other similar components, based on something I saw in a Stack Overflow question recently

```suggestion
The Snackbar component appears temporarily and floats above the UI to provide users with (non-critical) updates on an app's processes.
```